### PR TITLE
fix some bugs about rescheduling plugin

### PR DESCRIPTION
**Pull request by Thor-wl**

### CODE REVIEW

File: pkg/scheduler/actions/shuffle/shuffle.go [hunk 1]
```diff
@@ -57,6 +57,9 @@ func (shuffle *Action) Execute(ssn *framework.Session) {
 			}
 		}
 	}
+	for _, task := range tasks {
+		klog.V(3).Infof("task: %s\n", task.Name)
```
Comment (wpeng102) [inline pkg/scheduler/actions/shuffle/shuffle.go]: Not sure if this will lead huge logs

Comment (Thor-wl) [reply pkg/scheduler/actions/shuffle/shuffle.go]: OK, let me update the level to v4.

File: pkg/scheduler/plugins/rescheduling/low_node_utilization.go [hunk 2]
```diff
@@ -58,15 +58,34 @@ func (lnuc *LowNodeUtilizationConf) parse(configs map[string]interface{}) {
 		return
 	}
 	lowThresholdsConfigs, ok := configs["thresholds"]
+	klog.V(3).Infof("lowThresholdsConfigs: %v, ok: %t\n", lowThresholdsConfigs, ok)
 	if ok {
-		lowConfigs, _ := lowThresholdsConfigs.(map[string]int)
-		parseThreshold(lowConfigs, lnuc, "Thresholds")
+		lowConfigs, ok := lowThresholdsConfigs.(map[interface{}]interface{})
+		if !ok {
+			klog.V(3).Infoln("convert lowConfigs error")
```
Comment (wpeng102) [inline pkg/scheduler/plugins/rescheduling/low_node_utilization.go]: suggest use warning level

File: pkg/scheduler/plugins/rescheduling/low_node_utilization.go [hunk 3]
```diff
@@ -58,15 +58,34 @@ func (lnuc *LowNodeUtilizationConf) parse(configs map[string]interface{}) {
...
+		}
+		klog.V(3).Infof("lowConfigs: %v, ok: %t\n", lowConfigs, ok)
 	}
 	targetThresholdsConfigs, ok := configs["targetThresholds"]
 	if ok {
-		targetConfigs, _ := targetThresholdsConfigs.(map[string]int)
-		parseThreshold(targetConfigs, lnuc, "TargetThresholds")
+		targetConfigs, ok := targetThresholdsConfigs.(map[interface{}]interface{})
+		if !ok {
+			klog.V(3).Infoln("convert targetConfigs error")
```
Comment (wpeng102) [inline pkg/scheduler/plugins/rescheduling/low_node_utilization.go]: suggest use warning level

File: pkg/scheduler/plugins/rescheduling/low_node_utilization.go [hunk 4]
```diff
@@ -59,13 +59,29 @@ func (lnuc *LowNodeUtilizationConf) parse(configs map[string]interface{}) {
 	}
 	lowThresholdsConfigs, ok := configs["thresholds"]
 	if ok {
-		lowConfigs, _ := lowThresholdsConfigs.(map[string]int)
-		parseThreshold(lowConfigs, lnuc, "Thresholds")
+		lowConfigs, ok := lowThresholdsConfigs.(map[interface{}]interface{})
+		if !ok {
+			klog.Errorln("convert lowConfigs error")
```
Comment (william-wang) [inline pkg/scheduler/plugins/rescheduling/low_node_utilization.go:64]: Volcano log is a complete sentence. Please update it.

File: pkg/controllers/job/plugins/distributed-framework/mpi/mpi.go [hunk 5]
```diff
@@ -28,14 +28,19 @@ import (
 )
 
 const (
-	MpiPluginName = "mpi"
-	DefaultPort   = 22
+	// MPIPluginName is the name of the plugin
```
Comment (william-wang) [inline pkg/controllers/job/plugins/distributed-framework/mpi/mpi.go:31]: Please seperate the mpi related refactor to another pr. It's inconsist with title.

Comment (Thor-wl) [reply pkg/controllers/job/plugins/distributed-framework/mpi/mpi.go:31]: The update comes from the CI check - Code Verify. If not included in this PR, this CI item will fail.

File: pkg/scheduler/plugins/rescheduling/low_node_utilization.go (hunk 4 above)
Comment (Thor-wl) [reply pkg/scheduler/plugins/rescheduling/low_node_utilization.go:64]: OK.

File: pkg/scheduler/plugins/rescheduling/low_node_utilization.go (hunk 3 above)
Comment (Thor-wl) [reply pkg/scheduler/plugins/rescheduling/low_node_utilization.go]: OK

File: pkg/scheduler/plugins/rescheduling/low_node_utilization.go (hunk 2 above)
Comment (Thor-wl) [reply pkg/scheduler/plugins/rescheduling/low_node_utilization.go]: OK

File: pkg/scheduler/actions/shuffle/shuffle.go [hunk 6]
```diff
@@ -61,7 +61,7 @@ func (shuffle *Action) Execute(ssn *framework.Session) {
 	// Evict target workloads
 	victims := ssn.VictimTasks(tasks)
 	for victim := range victims {
-		klog.V(5).Infof("Victim %s: [ns: %s, job: %s]\n", victim.Name, victim.Namespace, victim.Job)
+		klog.V(3).Infof("Victim %s: [ns: %s, job: %s]\n", victim.Name, victim.Namespace, victim.Job)
```
Comment (william-wang) [inline pkg/scheduler/actions/shuffle/shuffle.go:64]: Make the log to be a complete sentence and easy to read for users.  e.g. "Job %s from namespace %s will be evited\n."

Comment (Thor-wl) [reply pkg/scheduler/actions/shuffle/shuffle.go:64]: OK.

File: pkg/controllers/job/plugins/distributed-framework/mpi/mpi.go (hunk 5 above)
Comment (william-wang) [reply pkg/controllers/job/plugins/distributed-framework/mpi/mpi.go:31]: OK, I see.

**Review by wpeng102 — COMMENTED** [review]
/lgtm

**Review by william-wang — APPROVED** [review]
/approve

**Pull request merged**